float64 Gripper
float64 Joint1
